### PR TITLE
Separate language mapping

### PR DESCRIPTION
**Pull request by amrondonp**
Separate language mapping

### CODE REVIEW

File: inginious/frontend/common/static/js/task.js [hunk 1]
```diff
@@ -995,8 +995,8 @@ function changeSubmissionLanguage(problemId){
     editor.updateLintStatus([]);
 }
 
-function getLanguageForProblemId(problemId){
-    var codemirrorLanguages = {
+function codemirrorLanguage(backEndLanguage) {
```
Comment (0xDCA) [inline inginious/frontend/common/static/js/task.js]: Please use a verb

**Comment by 0xDCA** [comment]
LGTM

**Review by larranaga — APPROVED** [review]
LGTM

**Review by lin99 — APPROVED** [review]
looks good

**Review by 0xDCA — APPROVED** [review]

**Pull request merged**